dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/icon/IconController.java

```diff
@@ -143,4 +161,42 @@ public void getIconData(HttpServletResponse response, @PathVariable String iconK
     webMessage.setMessage(String.format("Icon %s deleted", iconKey));
     return webMessage;
   }
+
+  private void downloadDefaultIcon(String defaultIcon, HttpServletResponse response)
```

Comment:
These methods are new and I don't see any new tests for it. I assume it is not tested but should be.